src/v3/src/components/ConsentHeader/ConsentHeader.tsx

```diff
@@ -85,7 +71,7 @@ const ConsentHeader: FunctionComponent = () => {
         display="flex"
         justifyContent="center"
       >
-        {typeof href !== 'undefined'
+        {typeof href !== 'undefined' && typeof logoHref !== 'undefined'
```

Comment:
is the logo always a link?  Can there be a logo with no href?

Reply:
Yep absolutely! When there is no clientUri (href for the logo), then it will fall to the `else` condition which will just try to generate the logo without an anchor, see: https://github.com/okta/okta-signin-widget/pull/3318/files#diff-ccfe8154ab15451a346edff46636e742d6c8fcfe8089731a928c6421704fc104R87

Reply:
ahh okay got it I missed that part, thank you!